cypress/specs/app.spec.ts

```diff
@@ -172,5 +172,12 @@ describe('App', () => {
         cy.matchImageSnapshot('logspectrum');
       }
     });
+
+    it('visualize dataset with malformed NeXus metadata', () => {
+      cy.findByRole('treeitem', { name: 'invalid' }).click();
+      cy.findByRole('treeitem', { name: 'default_not_found' }).click();
+
+      cy.findByText('Expected to find NXdata group at path "/test"');
+    });
```

Comment:
Adding one E2E test to make sure an error thrown in a `supportsEntity` function is indeed displayed.